string plyfile
---
uint32 count
